basics/uri.adoc

```diff
@@ -98,11 +158,96 @@ a| Method or topic Id from the protobuf file
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|  UP_VERSION   |      TYPE     |           URESOURCE_ID        |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                     UAUTHORITY_ADDRESS                        |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|        UENTITY_ID             |VER_MAJOR|     VER_MINOR       |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+----
+
+.IPv6
+[#ipv6-micro-uri,java]     
+----
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|  UP_VERSION   |      TYPE     |           URESOURCE_ID        |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
+|                     UAUTHORITY_ADDRESS                        |
+|                                                               |
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|        UENTITY_ID             |VER_MAJOR|     VER_MINOR       |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+----    
+
+* All fields *MUST* be populated
+* Fields are Big-Endian unless otherwise specified
+
+.Micro Uri Fields
+[#micro-uri-fields,width="100%",cols="20%,30%,50%",options="header"]
+|===
+|Field |Description | Requirements
+
+|`UP_VERSION` |Current version of this specification | *MUST* be 0x01
+
+|`TYPE` | Type of Micro Uri   a| *MUST* be one of the following values:
+
+!===
+!*Value* !*Description*
+!0x00 !Local
+!0x1 !IPv4
+!0x2 !IPv6
+!others !Reserved for future use
+!===
+
+|`UNUSED` |Reserved for future use | *MUST* be 0x00
+
+|`URESOURCE_ID` | The ID assigned to the topic in the proto (unique per uE) | 
+
+|`UAUTHORITY_ADDRESS` | UAuthority IP Address | *MUST* be a valid IP address
+
+|`UENTITY_ID` | UE Identifiers |
+
+|`VER_MAJOR` | UEntity MAJOR version (1-31) | *MUST* be a valid MAJOR version number (1 to 31)
+
+|`VER_MINOR` | UEntity MINOR version | *MUST* be a valid MINOR version number (0-1023)
+
+
+|===
 
 == Wildcards
-The wildcard character `*` is used in certain circumstances in lieu of an item in the URI for matching any character or sequence of characters in a search.
+wildcard are used to replace portions of the URI to perform pattern matching (i.e. subscribing to multiple topics, searching for services, etc...). 
+ 
+ * *SHOULD* support wildcards in the authority, entity, resource, and message portions of the URI
+ * Wildcards *MUST* be `*` (asterisk) for long and short form URIs
+ * Wildcard *MUST* be the max value for Micro URI (i.e. 0xFF for 8 bit item)
+
+NOTE: Not all uPlatforms and uEs support wildcards, please refer to the uPlatform and uE documentation for more information.
 
 == Example URIs
```

Comment:
Is it possible to extend the example URIs to show the short Uri and Micro Uri?

Reply:
yes done (in next merge)